modules/i3s/src/helpers/i3s-nodepages-tiles.js

```diff
@@ -48,11 +48,13 @@ export default class I3SNodePagesTiles {
     let materialDefinition = null;
     let textureFormat = 'jpeg';
     let attributeUrls = [];
+    let isDracoGeometry = false;
 
     if (node && node.mesh) {
-      if (node.mesh.geometry) {
-        contentUrl = `${this.tileset.url}/nodes/${node.mesh.geometry.resource}/geometries/0`;
-      }
+      const {url, isDracoGeometry: isDracoGeometryResult} = (node.mesh.geometry &&
```

Comment:
This looks like the wrong place to know about whether things are draco encoded or not. Can't this be handled and results "normalized" on the node level?

The expressions are a bit messy, if possible make more effort to write this cleanly.

Please add some comment.